tenzir/tests/ast/control_flow.tql

```diff
@@ -1,3 +1,5 @@
+// test: ast
+
```

Comment:
Can we still have this derived implicitly if not specified?

Reply:
We _can_ for sure, but I'm not convinced it's a good idea. You then lose the ability to just share a test file.

Reply:
The more I think about this the less I like it. If, in the future, we want to allow tests to be part of packages or third-party plugins or live in another repo, then having the directory structure affect how the test runs is a terrible idea.